django/forms/widgets.py

```
@@ -787,6 +787,19 @@ class CheckboxSelectMultiple(RendererMixin, SelectMultiple):
     renderer = CheckboxFieldRenderer
     _empty_value = []
 
+    def build_attrs(self, extra_attrs=None, **kwargs):
+        attrs = super(CheckboxSelectMultiple, self).build_attrs(extra_attrs, **kwargs)
+        # The required attribute comes from the bound field if the form field
+        # is required. When required is set on multiple checkboxes with the
+        # same name, the browser will require each checkbox be checked to
+        # submit the form. This is not the desired validation, so remove the
+        # attribute.
+        try:
+            del attrs['required']
+        except KeyError:
+            pass
```

I think the motto apply in the case where you let the exception propagate, when you catch it anyway I see no harm in using `attrs.pop('required', None)` instead.